ckanext/datastore/logic/schema.py

```diff
@@ -167,7 +169,9 @@ def datastore_search_schema():
         'plain': [ignore_missing, boolean_validator],
         'filters': [ignore_missing, json_validator],
         'language': [ignore_missing, text_type],
-        'limit': [ignore_missing, int_validator],
+        'limit': [default(100), natural_number_validator,
```

Comment:
This is an API change that might break assumptions in external scripts, we should call it out in the docs and make it configurable too.

Reply:
Resolved in https://github.com/ckan/ckan/pull/4562/commits/51337ff02aaf1ac2f216563a4c0c5b1a27c5700a